### PR TITLE
Added .wav file link

### PR DESCRIPTION
**Pull request by wiazur**
## Purpose

* There was no .wav file included in this sample, so I provided a link to the weather .wav file from another sample. The user might not remember where that was exactly to look for it themselves.

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[x ] Documentation content changes
[ ] Other... Please describe:
```

### CODE REVIEW

**Comment by mahilleb-msft** [comment]
Thanks @wiazur for this and the other PRs .. We're going to look at them as soon as possible...

File: samples/csharp/uwp/speechtotext-uwp/README.md [hunk 1]
```diff
@@ -40,6 +40,7 @@ The app displays a graphical user interface (GUI).
 > **Note:**
 > if you are using your own `.wav` file as input source, make sure it is in the right format.
 > Currently, the only supported `.wav` format is **mono (single-channel), 16 kHz sample rate, 16 bits per sample**.
+> You may also use the [weather .wav file](https://github.com/Azure-Samples/cognitive-services-speech-sdk/tree/master/samples/csharp/dotnet-windows/speechtotext-wpf/speechtotext-wpf/audio) in the Speech samples.
```
Comment (mahilleb-msft) [inline samples/csharp/uwp/speechtotext-uwp/README.md]: >https://github.com/Azure-Samples/cognitive-services-speech-sdk/tree/master/samples/csharp/dotnet-windows/speechtotext-wpf/speechtotext-wpf/audio [](start = 43, length = 144)

could you update the destination location to
`[https://raw.githubusercontent.com/Azure-Samples/cognitive-services-speech-sdk/f9807b1079f3a85f07cbb6d762c6b5449d536027/samples/cpp/windows/console/samples/whatstheweatherlike.wav`](https://raw.githubusercontent.com/Azure-Samples/cognitive-services-speech-sdk/f9807b1079f3a85f07cbb6d762c6b5449d536027/samples/cpp/windows/console/samples/whatstheweatherlike.wav`)
?
Thanks, Mark

**Comment by mahilleb-msft** [comment]
@wiazur - could you address the above comment? I'll merge this one then...

Sorry for the delay reviewing...

File: samples/csharp/uwp/speechtotext-uwp/README.md (hunk 1 above)
Comment (wiazur) [reply samples/csharp/uwp/speechtotext-uwp/README.md]: Hi Mark, I tried this new link in a browser but it says "This audio not supported". Does it work in other browsers? I tried it in Edge.
@mahilleb-msft

**Comment by wiazur** [comment]
Thanks Mark! I changed the link. It did not offer a right-click save-as option in Edge, but it worked in IE, Chrome and Firefox. 😊

**Pull request merged**